### ISSUE TITLE
how to setup and/or disable oauth

### ISSUE THREAD

**Issue by CybotTM** [netresearch/timetracker#11]
how to setup and/or disable oauth?

whether i get
```
Fatal error: Uncaught Symfony\Component\Config\Definition\Exception\InvalidConfigurationException: The path "hwi_oauth.resource_owners.jira.client_id" cannot contain an empty value, but got null.
```
or
```
Warning: file_get_contents(abc): failed to open stream: No such file or directory in vendor/hwi/oauth-bundle/HWI/Bundle/OAuthBundle/Security/OAuthUtils.php on line 216

Warning: openssl_sign(): supplied key param cannot be coerced into a private key in vendor/hwi/oauth-bundle/HWI/Bundle/OAuthBundle/Security/OAuthUtils.php on line 219

Warning: openssl_free_key() expects parameter 1 to be resource, boolean given in vendor/hwi/oauth-bundle/HWI/Bundle/OAuthBundle/Security/OAuthUtils.php on line 220
{"message":"Invalid Ticketsystem Token (Jira) you're going to be forwarded","forwardUrl":"\/connectjira\/jira"}
```

**Comment by CybotTM** [comment]
@nemesis621 can you help?

**Comment by nemesis621** [comment]
I've changed the OAuth behavoir in the latest mogic-le/timetracker:master. The Ticketsystem is no longer configured by paramater.yml, config.yml and service.yml. The necessary OAuth information can be managed via frontend at the manage Ticketsystem tab. This allows to use OAuth features on multiple ticketsystems. This requires a DB update (sql/upgrade-1.0.0.sql). Furthermore the hwi/HWIOAuthBundle and the ci/restclientbundle were removed from composer.
